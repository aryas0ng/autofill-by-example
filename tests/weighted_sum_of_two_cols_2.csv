2,4,18
1,0,5
2,3,16
3,8,31
10,10,70
5,5,
4,4,
19,4,
2,21,
1,21,
9,10,
7,7,
4,4,
3,3,
12,3,
4,16,
1,16,
26,27,
8,8,
3,3,
3,3,
8,11,
4,4,