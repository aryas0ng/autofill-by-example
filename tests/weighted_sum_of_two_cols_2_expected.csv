2,4,18
1,0,5
2,3,16
3,8,31
10,10,70
5,5,35
4,4,28
19,4,103
2,21,52
1,21,47
9,10,65
7,7,49
4,4,28
3,3,21
12,3,66
4,16,52
1,16,37
26,27,184
8,8,56
3,3,21
3,3,21
8,11,62
4,4,28